### PR TITLE
Create beherabikram

### PR DESCRIPTION
**Pull request by beherabikram**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8039"></a>

### CODE REVIEW

File: beherabikram [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Sara Jaoude",
```
Comment (sandramsc) [inline beherabikram:2]: Unless you are the real Sara Jaoude, I suggest you change the "name" value to **Bikram Behere** :)
```suggestion
  "name": "Bikram Behere",
```

File: beherabikram [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado"
```
Comment (sandramsc) [inline beherabikram:3]: There is a comma (**,**) missing on line 3, rendering the json file invalid.
```suggestion
  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado",
```

File: beherabikram [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  "displayStatsPublic": true,
+  "bio": "I am a MERNStack developer",
```
Comment (sandramsc) [inline beherabikram:5]: Duplicate **bio** tag. Consider removing this or the first one and combine the text 
```suggestion
  "bio": "I am a MERNStack developer. Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado",
```

File: beherabikram [hunk 4]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Sara Jaoude",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado"
+  "displayStatsPublic": true,
```
Comment (sandramsc) [inline beherabikram:4]: This tag is no longer needed, consider removing it.
nolongern
```suggestion
```

File: beherabikram [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  "bio": "I am a MERNStack developer",
+  "avatar": "https://avatars.githubusercontent.com/u/101280738?s=400&u=ab994845489a4322a33f2767f0ea7359f414b7b7&v=4",
```
Comment (sandramsc) [inline beherabikram:6]: This line is no longer needed, consider removing it.
```suggestion
```

File: beherabikram [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+       {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/beherabikram",
+      "icon": "FaGitHub"
+    },
+        {
+      "name": "Follow me on Linkdin",
+      "url": "https://www.linkedin.com/in/bikram-behera-4783b1259/",
+      "icon": "FaLInkdin"
+    },
```
Comment (sandramsc) [inline beherabikram:27]: Consider removing the additional comma (**,**) it renders, the json file invalid
```suggestion
    }
```

**Review by sandramsc — CHANGES_REQUESTED** [review]

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for your contribution 👍 

In addition to the inline comments the community have made
- you have 2 `bio` properties in the json file, these property names must be unique, so please remove 1 of the `bio`
- move the json file to the `data` directory as mentioned in the documentation

File: beherabikram [hunk 7]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Bikram Behere",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado"
```
Comment (SaraJaoude) [inline beherabikram:3]: this is the bio in the example docs, which is my bio so this should be removed completely:
```suggestion
```

File: beherabikram (hunk 3 above)
Comment (SaraJaoude) [reply beherabikram:3]: Given that the section `Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado"` is the Bio used in the example docs and my own personal bio I suggest leaving only `I am a MERNStack developer`

File: beherabikram [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+    },
+       {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/beherabikram",
+      "icon": "FaGitHub"
+    },
+        {
+      "name": "Follow me on Linkdin",
+      "url": "https://www.linkedin.com/in/bikram-behera-4783b1259/",
+      "icon": "FaLInkdin"
```
Comment (SaraJaoude) [inline beherabikram:23]: Please check the spelling for LinkedIn under `name` and `icon`:
```suggestion
      "name": "Follow me on LinkedIn",
      "url": "https://www.linkedin.com/in/bikram-behera-4783b1259/",
      "icon": "FaLinkedin"
```

File: beherabikram (hunk 2 above)
Comment (SaraJaoude) [reply beherabikram]: I will mark this conversation as resolved as this bio should be removed as it is the bio used in the example docs (and my personal bio).

**Comment by beherabikram** [comment]
I made mistake here, So i will try it again.

**Pull request closed without merging**